### DISCUSSION TITLE
Is yellow focus halo accessible for contrast ratio if item highlighted is on a light background?

### DISCUSSION THREAD

**Discussion by GnatalieH** [department-of-veterans-affairs/vets-design-system-documentation#528] [Announcements]
Example:

![image](https://user-images.githubusercontent.com/88347129/130149063-343dcb07-4be8-4b7b-ac7e-c896393f06ba.png)

**Comment by 1Copenut** [comment]
Our focus-halo yellow #f9c642 only has a 1.68:1 contrast ratio with the white background. We will need to make this a minimum 3:1 contrast ratio on light and dark backgrounds to meet [SC 1.4.11 Non-text Contrast Ratio](https://www.w3.org/TR/WCAG21/#non-text-contrast).

---
https://whocanuse.com/?b=ffffff&c=f6bf4b&f=20&s=

![Screen Shot 2021-08-20 at 10 44 13 AM](https://user-images.githubusercontent.com/934879/130259390-3086694f-e00a-4779-b655-5b1a741125f9.png)
